README.md

```diff
@@ -3,17 +3,49 @@ topic: sample
 services: azure-search
 name: Azure Search Sample Data
 description: |
-  Data and index definition metadata for use in the Azure Search samples.
+  Data files used in Azure Search quickstarts, tutorials, and examples.
 products:
   - azure-search
 ---
 
 # Azure Search Sample Data
 
-Data and index definition metadata for use in the Azure Search samples.  Use this as a starting point for exploring Azure Search functionality.
+Data files used in Azure Search quickstarts, tutorials, and examples. Use this data as a starting point for exploring Azure Search functionality.
 
-Each folder represents a different sample data set:
+Each folder represents a different sample data set.
 
 ## Hotels
 
-Data for samples that reference the Hotels schema and data set.  This is the default set for most Azure Search samples. 
+The Hotels sample data is used in quickstarts, tutorials, and examples. It consists of 50 hotels located in cities across the United States. The data includes images, hotel information, and room information. Data files are in JSON, but there are several versions depending on whether you are uploading it Azure Cosmos DB or pushing it to an index in Azure Search. 
+
+Alternatively, you can access this data from a read-only sandbox in the following ways. 
+
+### Hosted read-only index
+
+There is a staged index populated with the hotels demo data, hosted on an Azure Search service, accessed with a query api-key for read-only access to hotel documents.
+
+Endpoint to the index: `https://azs-playground.search.windows.net/indexes/hotels/docs?api-version=2017-11-11&search=*`
+
+query api-key: `EA4510A6219E14888741FCFC19BFBB82`
+
+### Hosted read-only version database
+
+The hotels demo data is also in Azure Cosmos DB. Using the **Import data** wizard, you can create a new Cosmos DB data source based on this connection information.
+
+Cosmos DB connection string: `AccountEndpoint=https://hotels-docbb.documents.azure.com:443/;AccountKey=4UPsNZyFAjgZ1tzHPGZaxS09XcwLrIawbXBWk6IixcxJoSePTcjBn0mi53XiKWu8MaUgowUhIovOv7kjksqAug==;`

```

Comment:
In order for this to work properly with indexers, the connection string needs to have the database name info attached to the end.  Not sure if the portal does this automatically or not. If the portal does not do this, then the connection string should be: "AccountEndpoint=https://hotels-docbb.documents.azure.com:443/;AccountKey=4UPsNZyFAjgZ1tzHPGZaxS09XcwLrIawbXBWk6IixcxJoSePTcjBn0mi53XiKWu8MaUgowUhIovOv7kjksqAug==;Database=SampleData"